cmd/signing.go

```diff
@@ -197,6 +199,10 @@ func SignDigest(c *cli.Context) error {
 
 	signingOutcomesChannel := make(chan *signingOutcome, len(signers))
 
+	pubKeyToAddressFn := func(publicKey cecdsa.PublicKey) []byte {
```

Comment:
Why do we want to define this function here and pass it to the `CalculateSignature`? Can't we define it inside `tss` package? In `tss` package we have functions related to member identification https://github.com/keep-network/keep-ecdsa/blob/3099d5f5e8902839b17d5c847437355825173854/pkg/ecdsa/tss/member.go#L14-L16
And I believe it does the same thing as we define here.

Reply:
If we define it inside `tss` we are tightly-coupling `tss` package with one particular way of calculating address from a public key. This way, `tss` will only work with one particular chain implementation, or a family of chain implementations using the same algorithm for pub key -> address conversion.

Reply:
In TSS we should use `MemberID` as a way to identify members. We're creating the ID from PublicKey: https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/node/node.go#L147
https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/ecdsa/tss/protocol_announce.go#L79
Isn't it currently the same what we try to introduce here?

Reply:
Additionally isn't `elliptic.Marshal(publicKey.Curve, publicKey.X, publicKey.Y)` already a way of generic and chain agnostic identification?

Reply:
I do agree, 

> In TSS we should use `MemberID` as a way to identify members. We're creating the ID from PublicKey:
> 
> https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/node/node.go#L147
> 
> https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/ecdsa/tss/protocol_announce.go#L79
> 
> Isn't it currently the same what we try to introduce here?

No, we are trying to log operator address, logging public key is not going to help, the information has to be very specific so that the operator can act and get the things right quickly. We are not changing the logic here, public key is still used as an identifier for all the communication. We just need to have a way to convert public key to address so that it can be printed to logs.

Reply:
> Additionally isn't `elliptic.Marshal(publicKey.Curve, publicKey.X, publicKey.Y)` already a way of generic and chain agnostic identification?

We use operator address as an operator identifier. This would work only for chains that create operator addresses as public keys in (quoting) _uncompressed form specified in section 4.3.6 of ANSI X9.62._ The decision about how public key is mapped to address should be left to the chain, that's why we insert the strategy here.